README.md

```diff
@@ -312,7 +312,68 @@ Actions are managed with an [LWW-Element-Set CRDT](https://en.wikipedia.org/wiki
 
 
 ## 4.3 Verifications
-*This section is still under development and will cover a CRDT for verified data types, including proof of ownership of an address or a specific NFTs*
+
+Verifications are mutual proofs of ownership between Farcaster accounts and external identities. Initially, Ethereum addresses are the only supported external identities, though other blockchains can be added later.
+
+Verifications must be opted in to by both the external identity and the Farcaster account. The mutual opt-in is implemented in two steps:
+
+1. Create a `VerificationClaim` object that contains the external address and the Farcaster account. Have the external identity sign a hash of the claim object.
```

Comment:
On nomenclature, the `VerificationClaim` sub-object in this case is not just a claim, it is actually a verifiable proof. 


We could reframe a Verification is a bi-directional proof between a Farcaster account and another entity, which is composed by first creating a directional proof from external entity to farcaster address, and then wrapping it another directional proof from farcaster address to entity. 

The wording above is a little rough, but I think this helps people understand that it is two way.